src/venueless-help-page.html

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Help Page
+description: Find Help
+---
+
+<div class="hero">
+  <div class="wrapper">
+    <header class="flex flex-wrap items-center gap-8 lg:gap-16">
+      <img
+        src="/assets/img/theme/hero-img.webp"
+        alt="welcome"
+        class="hidden w-full max-w-80 md:block" />
+      <div class="flex-1 max-w-screen-lg">
+        <h1 class="mb-8 pageheading">{{ title }}</h1>
+        <p class="subheading">
+          Need Help? This is the place! Or, the go to <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack</a> Page.
+        </p>
+      </div>
+    </header>
+  </div>
+</div>
+<!-- <div class="bg-gray-100 block-container">
+  <div class="wrapper">
+    <div class="flex items-center justify-center min-h-screen">
+      <div class="gap-8 list-disc lg:gap-16 md:columns-2 prose lg:prose-lg *:break-inside-avoid">
+          <p>General Venuless Questions?</p>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
+          <p>Technical Issues?</p>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
+          <p>Problems Accessing Slack?</p>
+          <p>Email Us!! <a href="mailto:{{site.contact_us_email}}">hello@djangocon.us</a> </p>
+          <p>Code of Conduct Issues?</p>
+          <p>Visit Our <a href="">Code of Conduct Page.</a></p>
+        </div>
+    </div>
+  </div>
+</div> -->
+<div class="block-container bg-blue">
+  <div class="wrapper">
+    <div class="grid gap-4 lg:grid-cols-4">
+        <div class="p-4 bg-gray-100">
+          <span class="text-xl font-bold">General Venueless Questions?</span>
+          <p class="">Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
+        </div>
+        <div class="p-4 bg-gray-200">
+          <span class="text-xl font-bold">Technical Issues?</span>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
```

Comment:
```suggestion
          <p>Come Visit Our Help Desk in <a href="https://djangoconus2024.slack.com">Slack </a></p>
```

Reply:
Will people who aren't already in slack be able to access it?

Reply:
Yes, anyone who has access to GitHub can see this page

Reply:
@blulady, no they will need to join via the link in the email they receive. However, once they are in the server, this URL will allow them to login with whatever credentials they set up.

Reply:
@tim-schilling Okay... If they already have a link.